integration-tests/tests/src/index.ts

```diff
@@ -46,5 +47,9 @@ describe(global.title, () => {
 
 afterEach(() => {
     // Remove all Realm files in the default directory
-    Realm.clearTestState();
+    if ("clearTestState" in Realm) {
+        Realm.clearTestState();
```

Comment:
Why don't you have to `(Realm as any).clearTestState()`?

Reply:
Because I updated the types of Realm JS to include the private `clearTestState` method :-)